mesheryctl/internal/cli/root/app/fixtures/onboard.application.response.golden

```diff
@@ -0,0 +1 @@
+[{"application_data":{"application_file":"name: TestMyApplication # display name in Meshery UI\nservices:\n  myapp:\n    type: Application\n    namespace: test\n    settings:\n      replicas: 1\n      containers:\n        - name: meshy\n          image: utkarsh23/meshy:v5\n          ports:\n            - name: http\n              containerPort: 8080\n"},"save":true}]
```

Comment:
@suhaniii add "save" in the name of the file, so that it is like `*.application.save.*` to be more specific

Reply:
@piyushsingariya you want me to do this?
``` 
[{"application_data":{"application_file":"name: MyApplicationSave
```

Reply:
nope the golden file name.

Reply:
Ok.. Updated